src/client/utils/hooks/useFeature.ts

```diff
@@ -0,0 +1,59 @@
+import { AppEnvironment } from 'src/shared/clientConfig'
+import { Market, useMarket } from 'components/utils/CurrentLocale'
+
+export enum Features {
+  OFFER_PAGE_INSURANCE_TOGGLE = 'OFFER_PAGE_INSURANCE_TOGGLE',
+  CHECKOUT_CREDIT_CHECK = 'CHECKOUT_CREDIT_CHECK',
+  TEST_FEATURE = 'TEST_FEATURE', // For unit testing purposes
+}
+
+interface FeatureConfig {
+  name: Features
+  envs: AppEnvironment[]
+  markets: Market[]
+}
+
+const Config: readonly FeatureConfig[] = [
+  {
+    name: Features.OFFER_PAGE_INSURANCE_TOGGLE,
+    envs: ['development', 'staging'],
+    markets: [Market.Se],
+  },
+  {
+    name: Features.CHECKOUT_CREDIT_CHECK,
+    envs: ['development', 'staging', 'production'],
```

Comment:
I'm thinking if we have a feature enabled in prod, maybe we could remove the envs field? So if you don't specify envs it's enabled in all envs. Also would be nice if you wouldn't need to specify both dev and staging since they should always be enabled in at the same time. All in all I think this is a great 🙌